code/modules/mob/dead/observer/observer.dm

```diff
@@ -895,3 +925,32 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 			client.images += t_ray_images
 		else
 			client.images -= stored_t_ray_images
+
+/mob/dead/observer/proc/show_rads()
```

Comment:
You shouldn't be duplicating the goggles code, refactor it to be cleaner if you want to do this